apps/src/templates/studioHomepages/GridContainer.jsx

```diff
@@ -17,12 +17,15 @@ const GridContainer = React.createClass({
   propTypes: {
     numColumns: React.PropTypes.number.isRequired,
     children: React.PropTypes.arrayOf(React.PropTypes.node),
-    isRtl: React.PropTypes.bool.isRequired
+    isRtl: React.PropTypes.bool.isRequired,
+    responsive: React.PropTypes.instanceOf(Responsive).isRequired
   },
 
   render() {
-    const { numColumns, isRtl } = this.props;
-    const itemWidth = styles.container.width / numColumns;
+    const { numColumns, isRtl, responsive } = this.props;
+
+    const nonResponsiveWidth = 100 / numColumns;
```

Comment:
This was a little confusing - can we add a comment that 100 is used as a percentage?